src/Data/Containers.hs

```diff
@@ -651,9 +794,15 @@ instance MonoZip LText.Text where
     {-# INLINE ounzip #-}
     {-# INLINE ozipWith #-}
 
+-- | Type class for maps that their keys can be converted
+-- into sets.
```

Comment:
Should maybe be "for maps whose keys" instead of "maps that their keys", or something along those lines